INSTALL.rst

```diff
@@ -1,6 +1,244 @@
 Installing scikit-image
 =======================
 
+We outline 4 different ways which we recommend using to get started with image
+processing and scikit-image.
+
+1. `Scientific Python Distributions`_ -- Recommended
+2. `pip based installation`_
+3. `conda based installation`_
+4. `System-wide package manager`_
+
+After your installation, you should be able to run the following code
+from a Python console or Jupyter notebook:
+
+.. code-block:: python
+
+  import skimage
+  print("scikit-image installed with version " + skimage.__version__)
+
+Should you have any questions getting started using ``scikit-image`` after
+reading this guide, please open an issue on
+`github <https://github.com/scikit-image/scikit-image/issues>`_
+to get additional help.
```

Comment:
This conflicts a bit with our advice to use SO, image.sc, or the mailing list for questions.

Reply:
I can cross reference the contact us section